lib/oban/crontab/cron.ex

```diff
@@ -83,6 +90,10 @@ defmodule Oban.Crontab.Cron do
   def parse!("@daily"), do: parse!("0 0 * * *")
   def parse!("@hourly"), do: parse!("0 * * * *")
 
+  def parse!("@reboot") do
+    struct!(__MODULE__, reboot: true)
+  end
```

Comment:
This fits nicely inline with the other irregular expressions:

```suggestion
  def parse!("@reboot"), do: struct!(__MODULE__, reboot: true)
```